Reconstruction/doc/RecCalorimeter.md

```diff
@@ -39,24 +36,36 @@ Energy calibration is done for sampling calorimeters to compensate for the energ
 
 ## 3. Noise addition
 
-Noise is added to all cells, not only those with signal. Hence a list of all cellIDs is needed and it is retrieved from the geometry. This is the only part which is geometry dependent and assumes tube geometry and phi-eta segmentation.
+Noise is added to all cells, not only those with signal. Hence a list of all existing in the geometry cell IDs is needed and it is retrieved from the geometry. This is the only part which is geometry dependent.
+
+Such a list is provided by a Gaudi tool deriving from `ICalorimeterTool`. Currently there are two implementations: `TubeLayerPhiEtaCaloTool` and `NestedVolumesCaloTool`.
```

Comment:
Add a sentence like "For other detector geometries additional dedicated tools could be implemented"?